vpdq/README.md

```diff
@@ -50,6 +50,7 @@ There are four inputs to the comparison algorithm, which determines if two video
 The query video’s frame PDQ hashes Q
 The comparison video’s frame PDQ hashes C
 The PDQ match distance D (example: 31)
+The PDQ quality filter tolerance F (example: 80), if either hash is below this quality level then they will not be compared
```

Comment:
blocking: I know I am a broken record on this point, but why are we suggestion a different value than what we are using in production? We should use the same value as we are going to use in the library, which I believe should be 50.

Reply:
Yes, it should be 50 instead of 80. Thanks for pointing it out.